_posts/2023-11-09-1.1.2-release.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Release 1.1.2"
+date:   2023-11-10 00:00:00 +0200
+tags: release
+author: lego11
+---
+
+Yet another update has been beamed to your computers now!
+This time starring:
+
+# New features
+
+## Automation improvements
+
+It is now possible to very easily build an automated station for APT or LRPT satellites, advantageously replacing previous setups that required anywhere from 5 to 12 separate pieces of software.
+
+lego11 has made [a detailed configuration guide](https://www.a-centauri.com/articoli/automating-satellite-reception-with-satdump) for such a station.
+
+Later, additional features such as simultaneous reception of NOAA APT and DSB, or LRPT and APT, are planned.
+
+
+## A lot of image enhancements
+
+We added **a lot of** enhancements for AVHRR this time.
+
+APT and HRPT had **all** the «traditional» NOAA enhancements added by lego11. Included are: BD (Hurricane), MD, MB, CC, EC, HE, HF, JF, JJ, ZA, TA. This means that SatDump has now complete feature parity with WXtoImg with respect to the meteorologic capabilities.
+
+Additionally, Marhmarchi has made an experimental night time fire detection enhancement based on the brightness temperature of AVHRR channels 3b and 4. It appears to work quite well, but more data is necessary.
+![](/assets/release_112/noaa_apt.png)  
+
+## Alpha channels in composites
+
+It is possible to have an alpha channel in the composite's equations, making it possible to build transparent composites that are designed to be overlaid on top of other channels.
+
+A typical use case would be overlaying MHS data over AVHRR, for example. 
+
+A `Rainfall (transparent)` composite that makes use of the alpha channel is already included in SatDump if you want to try it out.
+
+![Alt text](/assets/release_112/mhs.png)
+
+Projections are also supported with this new system.
+
+## New APT projections
+
+Aang23 remade the APT projections, which are now much better than with the previously used algorithm.
+
+## JPSS-1 DNB projections
+
+JPSS-1's DNB channel can now be projected correctly.
```

Comment:
Another category to add : JPSS/HRD now have RS checks enabled by default as data integrity matters more than a bit more data due to the compression.